js/client-static/rpc/index.ts

```diff
@@ -0,0 +1,86 @@
+import fetch from "node-fetch";
```

Comment:
Should Prolly pull this out to a seperate repo, test it up and what not, and import it.